dashboard/app/controllers/aichat_controller.rb

```diff
@@ -20,27 +22,20 @@ def chat_completion
     filter_result = ShareFiltering.find_failure(new_message_text, locale) if new_message_text
     # If the content is inappropriate, we skip sending to endpoint and instead hardcode a warning response on the front-end.
     return render(status: :ok, json: {status: filter_result.type, flagged_content: filter_result.content}) if filter_result
-    # TODO: Format input to send to Sagemaker.
+
+    input_json = AichatHelper.format_inputs_for_sagemaker_request(params[:aichatParameters], params[:storedMessages], params[:newMessage])
```

Comment:
Maybe something we can fix later, but `aichatParameters` is a bit confusing to me (aren't they all aichat params??)...`modelCustomizations` or something.

Reply:
Updated and also updated `chatContext`. Lmk what you think.